huaweicloud/services/rds/resource_huaweicloud_rds_account.go

```diff
@@ -0,0 +1,195 @@
+	}
+
+	// Split instance_id and user from resource id
+	parts := strings.SplitN(d.Id(), "/", 2)
+	if len(parts) != 2 {
+		return fmtp.DiagErrorf("invalid id format, must be <instance_id>/<user>")
+	}
+	instance_id := parts[0]
+	user_name := parts[1]
+
+	// items on every page, [1, 100]
+	limit := int32(100)
+	// List all db users
+	request := &model.ListDbUsersRequest{
+		InstanceId: instance_id,
+		Limit:      limit,
+		Page:       int32(1),
+	}
+
+	for {
+		response, err := client.ListDbUsers(request)
+		if err != nil {
+			return common.CheckDeletedError(d, err, "error listing RDS db users")
+		}
+		users := *response.Users
+		if len(users) == 0 {
+			break
+		} else {
+			request.Page += 1
+			for _, user := range users {
+				if user.Name == user_name {
+					d.Set("instance_id", instance_id)
+					d.Set("name", user_name)
+					return nil
+				}
+			}
+		}
+	}
+
+	// DB user deleted
+	d.SetId("")
+	log.Printf("[WARN] failed to fetch RDS db user %s: deleted", user_name)
+
+	return nil
+}
+
+func resourceRdsAccountUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	c := meta.(*config.Config)
+	client, err := c.HcRdsV3Client(c.GetRegion(d))
+	if err != nil {
+		return fmtp.DiagErrorf("error creating RDS client: %s", err)
+	}
+
+	// Split instance_id and user from resource id
+	parts := strings.SplitN(d.Id(), "/", 2)
+	if len(parts) != 2 {
+		return fmtp.DiagErrorf("invalid id format, must be <instance_id>/<user>")
+	}
+	instance_id := parts[0]
+	user := parts[1]
+
+	updateOpts := &model.SetDbUserPwdRequest{
+		InstanceId: instance_id,
+		Body: &model.DbUserPwdRequest{
+			Name:     user,
+			Password: d.Get("password").(string),
+		},
+	}
+
+	log.Printf("[DEBUG] Update RDS db user options: %#v", updateOpts)
+	_, err = client.SetDbUserPwd(updateOpts)
+	if err != nil {
+		return fmtp.DiagErrorf("error updating RDS db user: %s", err)
+	}
+
+	return resourceRdsAccountRead(ctx, d, meta)
+}
+
+func resourceRdsAccountDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	c := meta.(*config.Config)
+	client, err := c.HcRdsV3Client(c.GetRegion(d))
+	if err != nil {
+		return fmtp.DiagErrorf("error creating RDS client: %s", err)
+	}
+
+	// Split instance_id and user from resource id
+	parts := strings.SplitN(d.Id(), "/", 2)
+	if len(parts) != 2 {
+		return fmtp.DiagErrorf("invalid id format, must be <instance_id>/<user>")
+	}
+	instance_id := parts[0]
+	user := parts[1]
+
+	deleteOpts := &model.DeleteDbUserRequest{
+		InstanceId: instance_id,
+		UserName:   user,
+	}
+
+	log.Printf("[DEBUG] Delete RDS db user options: %#v", deleteOpts)
+	_, err = client.DeleteDbUser(deleteOpts)
+	if err != nil {
+		return fmtp.DiagErrorf("error deleting RDS db user: %s", err)
+	}
+
+	return nil
+}
+
+func resourceRdsAccountImportState(d *schema.ResourceData,
+	meta interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.SplitN(d.Id(), "/", 2)
+	if len(parts) != 2 {
```

Comment:
in the read function , those two parameter have be set . so this function can be deleted

Reply:
done